app/src/pages/trace/SessionDetails.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+import { css } from "@emotion/react";
+
+import { SessionDetailsQuery } from "./__generated__/SessionDetailsQuery.graphql";
+
+export type SessionDetailsProps = {
+  sessionId: string;
+};
+
+/**
+ * A component that shows the details of a session
+ */
+export function SessionDetails(props: SessionDetailsProps) {
+  const { sessionId } = props;
+  const data = useLazyLoadQuery<SessionDetailsQuery>(
+    graphql`
+      query SessionDetailsQuery($id: GlobalID!) {
+        session: node(id: $id) {
+          ... on ProjectSession {
+            traces {
+              edges {
+                trace: node {
+                  rootSpan {
+                    input {
+                      value
+                    }
+                    output {
+                      value
```

Comment:
yeah i see your point here, if there isn't a root span, this is gonna get nasty to deal with, can cross that bridge when we come to it i suppose